src/Connection.ts

```diff
@@ -0,0 +1,35 @@
+import { getConfig, createInstance } from './util/db';
```

Comment:
Remove `db.ts` altogether these functions could be made local methods to this class.